README.md

```diff
@@ -542,15 +542,17 @@ Static code analysis can be enabled by adding the following to the project file:
 <Project Sdk="MSBuild.Sdk.SqlProj/2.7.0">
   <PropertyGroup>
     <RunSqlCodeAnalysis>True</RunSqlCodeAnalysis>
-    <CodeAnalysisRules>-SqlServer.Rules.SRD0006;-Smells.*</CodeAnalysisRules>
+    <CodeAnalysisRules>-SqlServer.Rules.SRD0006;-Smells.*;+!SqlServer.Rules.SRN0005</CodeAnalysisRules>
```

Comment:
I would recommend against using the `+!` syntax in our only example. Instead, I'd keep this example the same as it was, and add a second example below showing how to force a rule violation. I think that would be clearer.